app/celery/scheduled_tasks.py

```diff
@@ -239,6 +249,7 @@ def beat_inbox_sms():
     to another list(list#2). The heartbeat will then call a job that saves list#2 to the DB
     and actually sends the sms for each notification saved.
     """
+    # TODO: ask what to do here
     receipt_id_sms, list_of_sms_notifications = sms_queue.poll()
```

Comment:
Right now we have 2 beat_inbox tasks, one for sms and one for email. We need 6: one for each redis queue, essentially. And instead of sending to `QueueNames.DATABASE` they will send to `QueueNames.PRIORITY_DATABASE`, etc (as in https://github.com/cds-snc/notification-api/pull/1533)

This work is in a different card though:
https://app.zenhub.com/workspaces/notify-planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/630
so it's out of scope for your PR.

Reply:
ya feel free to remove this! I will add this in my new PR.